docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.9"
+services:
+  litellm:
+    image: ghcr.io/berriai/litellm:main-v1.10.3

```

Comment:
If this is the default docker-compose it should point to latest

Reply:
Agreed, will update it.
Thanks!